recipes/git-url-parse/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "git-url-parse" %}
+{% set version = "1.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7b5f4e3aeb1d693afeee67a3bd4ac063f7206c2e8e46e559f0da0da98445f117
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pbr
+    - pip
+    - python
+  run:
+    - python
```

@beckermr, it is required...
```suggestion
    - python
    - pbr
```